Keypad/Keypad.module.less

```diff
@@ -0,0 +1,99 @@
+// Keypad.module.less
+//
+@import "../styles/skin.less";
+@import "../styles/mixins.less";
+
+.keypad {
+	height: 510px;
+	width: 480px;
+}
+
+.applySkins({
+	.keypad {
+		.keyContainer {
+			display: flex;
+			justify-content: center;
+			width: @agate-keypad-keycontainer-width;
+		}
+
+		.label {
+			bottom: @agate-keypad-subtext-bottom;
+			position: absolute;
+		}
+	}
+});
+
+.button {
+	.applySkins({
+		&.key {
+			height: @agate-keypad-button-height;
+			width: @agate-keypad-button-height;
+		}
+
+		.bg {
+			background-color: @agate-keypad-button-bg-color;
+			border: none;
+			border-radius: @agate-keypad-button-height;
+		}
+
+		.client {
+			color: @agate-keypad-button-color;
+
+			.text {
+				font-size: @agate-keypad-text-font-size;
+				line-height: @agate-keypad-text-line-height;
+			}
+
+			.label {
+				font-size: @agate-keypad-sub-text-font-size;
+				line-height: @agate-keypad-sub-text-line-height;
+			}
+
+			.label {
+				color: @agate-keypad-button-sub-text-color;
+			}
```

Comment:
Why it's separate?

Reply:
Probably a mix up, since there were two of us who worked on this task. Thank you for noticing!